.github/workflows/coverage-linux.yml

```diff
@@ -3,23 +3,21 @@ name: Coverage Linux
 on:
   pull_request:
     types: [opened, synchronize, reopened, ready_for_review]
-    paths-ignore:
-      - '**.md'
-      - benchmark/**
-      - deps/*
-      - doc/**
-      - .github/**
-      - '!.github/workflows/coverage-linux.yml'
+    paths:
+      - lib/**/*.js
+      - src/**/*.cc
+      - src/**/*.h
+      - test/**/*
+      - .github/workflows/coverage-linux.yml
```

Comment:
Theoretically changes to the Python test runner and/or build files (Makefile, gyp files) could also affect coverage results.